resources/docker-compose-regtest-liquid.yml

```diff
@@ -164,14 +143,4 @@ services:
       - electrs-liquid
     ports:
       - ${LIQUID_CHOPSTICKS_PORT}:3000
-    networks:
-      local:
-        ipv4_address: 10.10.0.17
     restart: unless-stopped
-
-networks:
-  local:
-    driver: bridge
-    ipam:
-      config:
-        - subnet: 10.10.0.0/24
```

Comment:
instead of removing the custom network, it would be nice to create one with the default driver and giving it a proper name like:

```
networks:
  default:
    name: nigiri
```

This requires changing the very first line of the yaml as:

```
version: '3.5'
```

This requires docker-compose's version to be at least [1.18](https://forums.docker.com/t/set-default-network-name-for-compose/36779/3) but I think it's fine since the latest one is 1.29.2. 

This way, we nigiri services would run in an isolated network, and attaching other services would be as easy as using the `nigiri` network (now the name was really ugly `resources_local`).

Reply:
The same change should be made to the `docker-compose-regtest.yml` file obviously.

Reply:
Sounds good and works as expected see commit [`ccfe5b3` (#116)](https://github.com/vulpemventures/nigiri/pull/116/commits/ccfe5b3e9448d7e58d590994a30ea17e9bc0561e)